build.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+jobs:
+    - job: 'tests_and_checks'
+
+      pool:
+          vmImage: 'macOS-10.13'
+
+      steps:
+          - task: NodeTool@0
+            inputs:
+                versionSpec: '10.15.0'
+            displayName: use node 10.15.0
+
+          - task: Npm@1
+            inputs:
+                command: custom
+                customCommand: ci
+            displayName: npm ci
+
+          - task: Npm@1
```

Comment:
`npm pack` is set up to run build/lint/format-check as a prerequisite already, and will produce the thing we'll actually need to publish as a build artifact for later publishing; consider just running that instead of running the individual steps here, to reduce redundancy between this file and package.json